contracts/src/v0.8/keystone/KeystoneFeedsConsumer.sol

```diff
@@ -2,17 +2,15 @@
 pragma solidity ^0.8.19;
 
 import {IReceiver} from "./interfaces/IReceiver.sol";
-import {ConfirmedOwner} from "../shared/access/ConfirmedOwner.sol";
+import {OwnerIsCreator} from "../shared/access/OwnerIsCreator.sol";
```

Comment:
We should drop this contract or merge it with the mock one we have for tests.

Reply:
It's not just an example contract and will be used by CCIP, maybe we can drop the mock instead

Reply:
@archseer it actually won't be used as-is by CCIP on prod, I was wrong about that.
However, I'd rather still keep it here as an example template on how to implement a basic consumer.